resources/linux/org.lite_xl.lite_xl.appdata.xml

```diff
@@ -29,6 +29,6 @@
   </provides>
 
   <releases>
-    <release version="2.1.1" date="2022-12-29" />
+    <release version="2.1.2" date="2022-12-29" />
```

Comment:
Commenting here as a reminder that this date will need to be updated before release.